web-common/src/components/data-graphic/constants/types.d.ts

```diff
@@ -1,4 +1,4 @@
 export interface DomainCoordinates {
-  x: number | Date;
-  y: number | Date;
+  x?: number;
+  y?: number;
```

Comment:
note that
(1) I could not find any instances of Date being used in this type (it always seems to be used in numeric operations, but please double check me on this)
(2) these fields are very commonly set to undefined by way of the `DEFAULT_COORDINATES` const which is used all over the place

```
export const DEFAULT_COORDINATES: DomainCoordinates = {
  x: undefined,
  y: undefined,
};
```